src/app/nav-bar/nav-bar.component.css

```diff
@@ -0,0 +1,48 @@
+.nav-container {
```

Comment:
use component wrapper. as example, wrap all of this code into nav-bar {} so in this way you're sure this css will be applied only for this component even if you have the same css classes in other places